### ISSUE TITLE
CSV save not implemented

### ISSUE THREAD

**Issue by LoganDark** [zodiacon/ProcMonXv2#12]
![image](https://github.com/zodiacon/ProcMonXv2/assets/4723091/71389593-b902-47e5-a868-4472aa3966ea)

this is due to

https://github.com/zodiacon/ProcMonXv2/blob/e5486fb473fcce8a1cc387b06db41bb17b12488a/ProcMonX/CSVEventDataSerializer.cpp#L11-L13

the CSV serializer being totally unimplemented

it's impossible for me to properly process this data inside ProcMonXv2 itself :/

**Comment by zodiacon** [comment]
You can select all, and right-click -> Copy and then paste into a text editor for now.

**Comment by LoganDark** [comment]
> You can select all, and right-click -> Copy and then paste into a text editor for now.

yeah when I do this the program just freezes. keep in mind this is enough data that the pmx file is 70 megabytes

it's also enough data that when I try to open the pmx in any object viewer, then *that* also freezes

so I really, truly do need the CSV export to do anything with this

(this isn't a demand, I'm just saying there's currently no workaround)

**Comment by zodiacon** [comment]
I agree, no doubt about that. There are just 24 hours a day...
